src/components/Navigation.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styles from "./Navigation.module.css";
+import {NavLink, useNavigate} from "react-router-dom";
+
+const Navigation = ({auth, setAuth}) => {
+
+    const navigate = useNavigate()
```

Comment:
Je hebt de navigatiebalk zo gemaakt dat alles blijft staan en er alleen blogoverzicht en blogpost bij komt als je bent ingelogd. In de uitwerkingen staat alleen home en login als je nog niet bent ingelogd. En na inloggen zie je alleen home, overzicht en de uitlog knop. Dit kan je op de volgende manier doen. De onderstaande code werkt niet, maar is alleen om te laten zien hoe je het moet opbouwen. Zie de uitwerkingen voor de werkende code.  Zelf had ik het ook niet zo mooi als in de uitwerkingen opgeschreven.
```suggestion
    
    {isAuth === true ? 
    <>
    <Navlink> overview </NavLink> 
    <Button> log out </Button>
    </>
    :
    <NavLink>Login</Navlink>
    }
    
```